niji/package.json

```diff
@@ -24,7 +24,20 @@
   },
   "homepage": "https://github.com/metakirby5/niji#readme",
   "dependencies": {
+    "bcrypt": "^0.8.7",
```

Comment:
Let's trim down the dependencies and only include them when we use them. For example. I don't see you using rxjs yet, so take it out.